isis/tests/FileNameTests.cpp

```diff
@@ -84,13 +84,21 @@ TEST(FileName, Extension) {
   EXPECT_EQ("cub", file.extension());
 }
 
-TEST(FileName, Expanded) {
+TEST(FileName, Expanded1) {
   QString relativeFileName("test.cub");
   FileName file("$ISISROOT/" + relativeFileName);
   QString isisRoot(getenv("ISISROOT"));
   EXPECT_EQ(isisRoot + "/" + relativeFileName, file.expanded());
 }
 
+TEST(FileName, Expanded2) {
```

Comment:
nitpick, but maybe rename these to be more like "ExpandedDefault" and "ExpandedUnderscore" which is more in line with how we would want to be naming ctests.